garde/tests/rules/length.rs

```diff
@@ -42,9 +42,9 @@ fn length_invalid() {
 
 #[derive(Debug, garde::Validate)]
 struct Exact<'a> {
-    #[garde(length(min = 2, max = 2))]
+    #[garde(length(equal = 2))]
     field: &'a str,
-    #[garde(inner(length(min = 2, max = 2)))]
+    #[garde(inner(length(equal = 2)))]
```

Comment:
For the sake of backwards compatibility, we should ensure that `length(min=N,max=N)` and `length(equal=N)` have the exact same result. Can you add a test for that?

```rust
#[derive(Debug, garde::Validate)]
struct MinMaxEqual<'a> {
  #[garde(length(min = 2, max = 2))]
  min_max: &'a str,
  #[garde(length(equal = 2))]
  equal: &'a str,
}
```

Reply:
I'll have a look yeah